templates/angular/ts/column-chart/default/files/src/app/components/__path__/__name__.component.ts

```diff
@@ -0,0 +1,86 @@
+import { Component } from "@angular/core";
+import { ActivatedRoute, Routes } from "@angular/router";
```

Comment:
These imports are not really needed for the component, right?

Reply:
Removed...